CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Add conf and env var to disable domain middlware to reduce overhead (but loosing some info in logs)
```

Comment:
```suggestion
- Add: conf and env var (DISABLE_DOMAIN_MIDDLEWARE) to disable domain middleware to reduce overhead (but loosing some info in logs)
```

No issue associated?

Reply:
done in https://github.com/telefonicaid/fiware-pep-steelskin/pull/508/commits/36c167765bf2120ddecf82d3668af636a958b913